### DISCUSSION TITLE
DDEV v1.21.3 PHP 8.2 - xdebug problems

### DISCUSSION THREAD

**Discussion by BartoszWoszczak** [ddev/ddev#4900] [Q&A] [answered]
Hey, do you have any problem with xDebug?

`PHP Warning:  Failed loading Zend extension 'xdebug.so' (tried: /usr/lib/php/20220829/xdebug.so (/usr/lib/php/20220829/xdebug.so: cannot open shared object file: No such file or directory), /usr/lib/php/20220829/xdebug.so.so (/usr/lib/php/20220829/xdebug.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
xdebug disabled`

I tried to fix this but nothing helps here, after restarting container is always broking.

**Comment by rfay** [comment, marked as answer]
Hi @BartoszWoszczak - Why are you using v1.21.3, could you please use current v1.21.6? 

PHP 8.2 didn't get all its PHP extensions (including xdebug) until DDEV v1.21.4

In general, please try using the current version of any tool during your debugging process.

**Reply by BartoszWoszczak** [reply]
Hey, I try to upgrade by `brew` on Mac but the version wasn't updated. I run the script and right now it's an upgraded version. I reinstalled containers and images and we will see. If everything will be ok, then I will close the discussion.

**Reply by rfay** [reply]
`ddev --version` will show you what version you have. `which ddev` will show you where it's coming from. You probably have a version of DDEV which was installed by something other than homebrew. If you're on an Apple Silicon mac (m1/m2) then `ls -l $(which ddev)` should show /opt/homebrew/bin/ddev`. If you're on an intel mac, it should show /usr/local/bin/ddev. In both cases you should see a symlink.

**Reply by BartoszWoszczak** [reply]
> `ddev --version` will show you what version you have. `which ddev` will show you where it's coming from. You probably have a version of DDEV which was installed by something other than homebrew. If you're on an Apple Silicon mac (m1/m2) then `ls -l $(which ddev)` should show /opt/homebrew/bin/ddev`. If you're on an intel mac, it should show /usr/local/bin/ddev. In both cases you should see a symlink.

` I run the script and right now it's an upgraded version` 

Ok, everything starts working, thx for the info and I'm closing the discussion.
